pkg/mount/sharedsubtree_linux.go

```diff
@@ -67,5 +67,5 @@ func ensureMountedAs(mountPoint, options string) error {
 		return err
 	}
 
-	return ForceMount("", mountPoint, "none", options)
+	return mount("", mountPoint, uintptr(0), options)
```

Comment:
This is failing;

```
00:41:38 pkg/mount/sharedsubtree_linux.go:70:31:warning: cannot use uintptr(0) (constant 0 of type uintptr) as string value in argument to mount (gosimple)
00:41:38 pkg/mount/sharedsubtree_linux.go:70:50:warning: too few arguments in call to mount (unconvert)
00:41:38 pkg/mount/sharedsubtree_linux.go:70:43:warning: cannot use options (variable of type string) as uintptr value in argument to mount (unconvert)
00:41:38 pkg/mount/sharedsubtree_linux.go:70:31:warning: cannot use uintptr(0) (constant 0 of type uintptr) as string value in argument to mount (unconvert)
```

Reply:
Thanks, I was too soon to push this. Please see the new commits.